tests/test_plotting.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import logging
+import os
+import pathlib
+
+import pandas as pd
+import pytest
+
+from tot.benchmark import SimpleBenchmark
+from tot.datasets.dataset import Dataset
+from tot.evaluation.metrics import ERROR_FUNCTIONS
+from tot.models.models_naive import NaiveModel, SeasonalNaiveModel
+from tot.models.models_neuralprophet import NeuralProphetModel
+from tot.models.models_simple import LinearRegressionModel, ProphetModel
+from tot.plotting import plot_plotly
+
+log = logging.getLogger("tot.test")
+log.setLevel("WARNING")
+log.parent.setLevel("WARNING")
+
+DIR = pathlib.Path(__file__).parent.parent.absolute()
+DATA_DIR = os.path.join(DIR, "datasets")
+PEYTON_FILE = os.path.join(DATA_DIR, "wp_log_peyton_manning.csv")
+AIR_FILE = os.path.join(DATA_DIR, "air_passengers.csv")
+ERCOT_FILE = os.path.join(DATA_DIR, "ercot_load_reduced.csv")
+SAVE_DIR = os.path.join(DIR, "tests", "test-logs")
+if not os.path.isdir(SAVE_DIR):
+    os.makedirs(SAVE_DIR)
+
+
+NROWS = 128
+EPOCHS = 2
+BATCH_SIZE = 64
+LR = 1.0
+ERCOT_REGIONS = ["NORTH", "EAST", "FAR_WEST"]
+PLOT = False
+
+# plot tests cover both plotting backends
+decorator_input = ["plotting_backend", ["plotly-auto", "plotly", "plotly-resampler"]]
```

Comment:
As everything is plotly here, how about renaming `plotly-auto` to `auto`? It just seemed to me at first glance, like the default would be `plotly` and `plotly-auto` and `plotly-resampler` were variants of it. Or how about `None` instead of `auto` (which is purely stylistic choice, for typing only strings might even be better)?

Reply:
Yes, I agree. I will go with None